### PR TITLE
Use new mechanism to opt-out the render message in rmarkdown

### PR DESCRIPTION
**Pull request by cderv**
require rmarkdown >= 2.7.6

This is to merge only once rmarkdown 2.8 is on CRAN so that we do not constrain **bookdown** to a rmarkdown version for new release. 

So I put that as a draft and we'll merge later.

### CODE REVIEW

**Review by yihui — APPROVED** [review]
Looks good. Let's merge it after the next rmarkdown release.

File: R/site.R [hunk 1]
```diff
@@ -34,13 +34,11 @@ bookdown_site = function(input, ...) {
     if (is.null(input_file)) {
       in_dir(input, render_book_script(output_format, envir, quiet))
     } else {
-      # Input file can be in a sub-directory  of the project. So we need to
-      # to emit our own message with absolute path
+      opts = options('rmarkdown.rstudio.preview' = FALSE)
```
Comment (yihui) [inline R/site.R:37]: Cosmetic...

```suggestion
      opts = options(rmarkdown.rstudio.preview = FALSE)
```

**Review by yihui — APPROVED** [review]
This PR is ready to go now.

**Comment by cderv** [comment]
Thanks!

**Pull request merged**